src/components/button-toggle-group/button-toggle-group.component.tsx

```diff
@@ -45,33 +47,46 @@ export interface ButtonToggleGroupProps
   fullWidth?: boolean;
   /** The alignment for the text in the label. */
   labelAlign?: "left" | "right";
-  /** Callback triggered by blur event on the input. */
-  onBlur?: (ev: React.FocusEvent<HTMLInputElement>) => void;
-  /** Callback triggered by change event on the input. */
-  onChange?: (ev: React.ChangeEvent<HTMLInputElement>) => void;
-  /** The value of the Button Toggle Group */
+  /** Callback triggered by blur event on one of the child buttons. */
+  onBlur?: (ev: React.FocusEvent<HTMLButtonElement>) => void;
```

Comment:
Is it necessary to still have `onBlur` on the `ButtonToggleGroup` considering that it is no longer meant to be used as a form input and no longer supports validations?

Reply:
Probably not necessary, no - but again I wanted to avoid removing props if there was some sensible way to keep them. HTML buttons, just like HTML buttons, can be blurred - I agree this is unlikely to be useful in a non-form context but you never know what consumers might be doing. Maybe this should be deprecated for now?